src/lib/globalState/scaleStore.ts

```diff
@@ -0,0 +1,7 @@
+import { writable, type Writable } from "svelte/store";
+
+/**
+ * Dines the current scaling of the main SVG view.
```

Comment:
what does it mean to Dine is it like serving but posh?

Reply:
Whoops. Defines 😂